exercises/sum-of-multiples/instructions.md

```diff
@@ -0,0 +1,27 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the number of the level that the player completed
```

Comment:
```suggestion
- the level (as number) that the player completed
```

I think from the rest of the exercise, the level _is_ numeric and the number is not a property of the level. If it is, then the sentence is, for me, hard to parse.

A bit futher down it says "level number" which also works, but is less suited for this sentence.